doc/api/worker_threads.md

```diff
@@ -242,6 +271,26 @@ new Worker('process.env.SET_IN_WORKER = "foo"', { eval: true, env: SHARE_ENV })
   });
 ```
 
+## `worker.setEnvironmentData(key[, value])`
+<!--YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `key` {any} Any arbitrary, cloneable JavaScript value that can be used as a
+  {Map} key.
```

Comment:
It seems like objects/arrays are undesirable keys?  Although they are cloneable and valid Map key's they will not work well for this since you cannot `environmentData.get(objectKey)` from new threads.  I'm not sure if/how the documentation should reflect this or if object/array keys should be rejected by `worker.setEnvironmentData()`.

Reply:
Possibly. I did not want to arbitrarily restrict those here, however. If there's agreement that the key types should be restricted, however, I certainly can do so.